src/main/java/com/airlines/model/AirplaneModel.java

```diff
@@ -0,0 +1,8 @@
+package com.airlines.model;
+
+public enum AirplaneModel {
+    Boeing737_800,
+    SSJ_95,
+    Tu_134,
```

Comment:
Enum is container for constant values. Constant values have name convention:
ALL_CAPITAL_LETTERS with rare exception.
so
BOEING_737_800
TU_134
and so on

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/bc4e257c81648513d3a0f4bc5de1c6e21696c76a